Package.swift

```diff
@@ -390,6 +408,96 @@ let package = Package(
       ]
     ),
 
+    .target(
+      name: "FirebaseFirestore",
+      dependencies: [
+        "FirebaseCore",
+        "leveldb",
+        .product(name: "nanopb", package: "nanopb"),
+        .product(name: "abseil", package: "abseil"),
+        .product(name: "gRPC-cpp", package: "gRPC"),
+      ],
+      path: "Firestore",
+      exclude: [
```

Comment:
Why are all these entries necessary? Shouldn't this only consist of things that would otherwise match the `sources` list?

Reply:
It's everything under Firestore that is not in `sources`. A better directory structure for SPM would be putting all sources in one directory.  I've sorted the list now.